ztp/Makefile

```diff
@@ -9,6 +9,16 @@ test-policygen:
 checkExtraManifests:
 	$(MAKE) -C ./extra-manifests-builder check
 
+checkSourceCRsAnnotation:
+	for sourcefile in ./source-crs/*.yaml;do \
+		if [[ "$${sourcefile}" != *"MachineConfig"* ]];then \
```

Comment:
Why not check MachineConfig files?

We have symlinks in source-crs for backwards compatibility, so it's possible users might still include them within as policy.

What happens if there's no annotation on a CR in a PGT?

Reply:
Based on the discussion this morning, no need waves for MCs we want to deprecate the use of MCs after installation via policies. We need to consider how to deprecate.

To the question that "What happens if there's no annotation on a CR in a PGT"
One policy should only have sources with same waves, otherwise, it's an error. This includes a CR doesn't have wave but others have.
If policy has no wave which means its sourceCRs do not have wave, this policy will not be managed by lifecycle operator.